plugins/cache.rst

```diff
@@ -36,18 +36,20 @@ control headers from the server as specified in :rfc:`7234`. It needs a
 Options
 -------
 
-The third parameter to the ``CachePlugin`` constructor takes an array of options. The plugin has 3 options you can
+The third parameter to the ``CachePlugin`` constructor takes an array of options. The plugin has four options you can
 configure. Their default values and meaning is described by the table below.
 
-+---------------------------+---------------+------------------------------------------------------+
-| Name                      | Default value | Description                                          |
-+===========================+===============+======================================================+
-| ``default_ttl``           | ``0``         | The default max age of a Response                    |
-+---------------------------+---------------+------------------------------------------------------+
-| ``respect_cache_headers`` | ``true``      | Whatever or not we should care about cache headers   |
-+---------------------------+---------------+------------------------------------------------------+
-| ``cache_lifetime``        | 30 days       | The minimum time we should store a cache item        |
-+---------------------------+---------------+------------------------------------------------------+
++---------------------------+---------------------+------------------------------------------------------+
+| Name                      | Default value       | Description                                          |
++===========================+=====================+======================================================+
+| ``default_ttl``           | ``0``               | The default max age of a Response                    |
++---------------------------+---------------------+------------------------------------------------------+
+| ``respect_cache_headers`` | ``true``            | Whatever or not we should care about cache headers   |
```

Comment:
s/Whatever/Whether/

(i see you only reformatted this, but just jumped into my eyes)